### DISCUSSION TITLE
Is calling the JS SDK from within pb_hooks recommended?

### DISCUSSION THREAD

**Discussion by ersinakinci** [pocketbase/pocketbase#4694] [Q&A]
The JS SDK has a lot of functionality. Does it make sense to create an instance, point it to localhost, and use that instead of the "native" hooks methods that aren't as well-documented?

So far, I've been sticking to `$app` and `$apis`, but it would be much faster to be able to copy/paste examples that use the SDK. Are there any edge cases that I should be aware of?

How would I even import the SDK?

**Comment by AxelRothe** [comment]
hey, you can write backend server code with the JS SDK signed in as an admin, with a pocketbase super admin account. you will need to handle authorisation yourself.

the problem with the sdk is, that it's running fetch requests, which are slow and add up vs. SQL queries that are lightning fast.

You also can't batch update or batch create or batch anything from the SDK. this can mean that some requests might take seconds to complete.
